packages/frontend/src/components/accounts/create/verify_account/VerifyWithEmailOption.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+
+import VerifyOption from './VerifyOption';
+
+export default ({
+    translateIdTitle,
+    translateIdDesc,
+    onClick,
+    activeVerificationOption,
```

Comment:
Not a blocker, but it'd be nice to keep the API for these components identical to the other VerifyOption components, and keep the magic string comparisons for passing in the correct value for `isActive` in the parent component.